tests/benchmarks/test_trt_llm_backend.py

```diff
@@ -0,0 +1,251 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This test is probably overkill too, I don't think it's worth the effort of maintaining this. Please remove.